src/main/java/ru/skypro/homework/manager/MyUserDetailsService.java

```diff
@@ -0,0 +1,64 @@
+package ru.skypro.homework.manager;
+
+
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Component;
+import ru.skypro.homework.entity.User;
+import ru.skypro.homework.mapper.UserMapper;
+import ru.skypro.homework.repository.UserRepository;
+
+@Component
+public class MyUserDetailsService implements UserDetailsService {
```

Comment:
```suggestion
public class MyUserDetailsService implements UserDetailsManager {
```
Нужно реализовать UserDetailsManager. Его использует spring.

Reply:
После этого придётся добавить метод `createUser(UserDetails user)`. Но он может оставаться пустым.
```
    @Override
    public void createUser(UserDetails user) {

    }
```